ocaml/tests/test_sm_features.ml

```diff
@@ -47,89 +47,87 @@ let test_sequences =
     {
       raw=
         [
-          "SR_PROBE"
```

Comment:
this does a lot more than remove the duplicate VDI_UPDATE

Reply:
Indeed. It is also what is on master. So you can approve this?